racket/src/ChezScheme/c/new-io.c

```diff
@@ -99,7 +99,7 @@ static int is_valid_lz4_length(iptr count);
   do { body;                                            \
     if (ok) { flag = 0; break; }                        \
     else {                                              \
-      INT errnum;                                       \
+      INT errnum = 0;                                   \
```

Comment:
Should do the same around lines 81 and 120?

Reply:
We can - although `gcc` didn't complain about those. I will check it anyway.